\chapter{Variant Detection}\label{var}
\section{Introduction}\label{var.introduction}
The mitochondrial genome is amenable to targeted resequencing by second-generation technologies.  Its relatively small size (16.5 \ac{kb}) and low repetitive sequence content make it much more likely that a unique sequence alignment can be determined for the short reads generated by current second-generation sequencing systems.

In order to identify the extent of mitochondrial genome variation in our sample set, we used two sequencing technologies for two distinct segments of the mitochondrial genome.  For the non-coding control region (position 16,024\--576), we performed bi-directional Sanger sequencing on all 419 cases and 415 controls.  A pooled Illumina Genome Analyzer (GA) sequencing strategy was used to identify variants in the entire mitochondrial genome.  See figure ~\ref{var.fig.design} for an outline of the variant detection strategy.

\begin{figure}
  \begin{center}
    \includegraphics[width=\textwidth]{fig/design.pdf}
  \end{center}
  \caption[Experimental Design for Variant Detection]{
    \small{\textbf{Experimental Design for Variant Detection.} Two sequencing technologies were used to identify mitochondrial genome variation.  The control-region was PCR-amplified in two segments and sequenced by Sanger sequencing in individuals.  The entire mitochondrial genome was PCR-amplified, pooled, and sequenced on the Illumina Genome Analyzer.  Coding region variants were carried forward to Sequenom genotyping in individuals.}}
  \label{var.fig.design}
\end{figure}

\section{Methods}\label{var.methods}
This study was approved by the joint Clinical Research Ethics Board of the British Columbia Cancer Agency and the University of British Columbia. All subjects gave written informed consent.

\subsection{Subjects and Samples}\label{methods.samples}
The subjects of this study were 419 healthy elderly individuals (cases) and 415 mid-life controls.  Cases were $> 85$ years old at the time of recruitment, and had not been diagnosed with cancer, cardiovascular disease, Alzheimer disease or diabetes.  Controls were 40-50 years old at recruitment, and were ascertained without regard to health status.  All participants are of European descent, based on subject-reported ethnicity of their four grandparents.  Total DNA was extracted from peripheral blood leukocytes using the Gentra Puregene Blood Kit (Qiagen), according to the manufacturer's protocol.

\subsection{Control Region PCR and Sanger Sequencing}\label{var.methods.control_region}
PCR primers were designed not to overlap with common polymorphic loci. In-silico PCR was performed using web service based at Kyushu University, to ensure that no nuclear DNA segments would be co-amplified\cite{kyushu}.  The mitochondrial control region was PCR-amplified with Platinum Pfx polymerase (Invitrogen).  PCR reactions were performed in 20 $\mu$L total volume containing: 20 ng template genomic DNA, 10 $\mu$M each of forward primer (MAP001\_F or MAP002.1\_F) and reverse primer (MAP001\_R or MAP002.1\_R) (Table ~\ref{var.table.primers}), 0.4 U Platinum Pfx enzyme, 10 mM each dNTPs, and 1x Phusion Buffer GC.  Forward and reverse primers incorporated the -21M13F (TGTAAAACGACGGCCAGT) and M13R (CAGGAAACAGCTATGAC) extensions, respectively, at their 5' ends.  Sequencing reactions were carried out as described previously \cite{brooks2004germline}.

\subsection{Sanger Sequence Assembly}
Sanger sequence traces were aligned to the \ac{rCRS} reference sequence (GenBank accession NC\_012920) with the Phred/Phrap/Consed suite, version 20.0 \cite{gordon1998consed,ewing1998base1,ewing1998base2}.  Polymorphisms were first detected automatically using Polyphred version 6.18. To minimize false-positives all non-reference alleles were manually confirmed by visual inspection of chromatograms by two people.

\subsection{Long PCR}\label{var.methods.long_pcr}
The mitochondrial genome was amplified using long-PCR with Phusion polymerase (Finnzymes).  PCR reactions were performed in 20 $\mu$L total volume containing: 20 ng template genomic DNA (2 ng/$\mu$L), 10 $\mu$M each of forward primer MAP011.1\_F and reverse primer MAP011.1\_R (Table ~\ref{var.table.primers}), 0.4 U Phusion enzyme, 10mM each dNTPs, and 1x Phusion Buffer GC.  The thermocycler program was: 1.) initial melt at 98\textdegree C for 30 seconds, 2.) melt at 98\textdegree C for 10 seconds, 3.) anneal/extend at  72\textdegree C for 8 minutes, 15 seconds 4.) repeat steps 2 and 3, 29 times 5.) final extension at 72\textdegree C for 10 minutes.

\begin{table}[htbp]
\begin{minipage}{\textwidth}
\caption[List of PCR Primers]{PCR primers used for Sanger sequencing and long-PCR.}
\label{var.table.primers}
\noindent\makebox[\textwidth]{%
\footnotesize
\begin{tabular}{l c l l}
Primer ID  & T$_{m}$ (\textdegree C) & Sequence & rCRS Position\\ \hline
MAP011.1-F & 66.3 & GGGAGCTCTCCATGCATTTGG      &     34-54 \\
MAP011.1-R & 64.7 & AGACCTGTGATCCATCGTGATGTC   & 16,558-12\\
MAP001-F   & 57.1 & (-21M13-Fwd\footnote{`-21M13-Fwd' = TGTAAAACGACGGCCAGT})GAAAAAGTCTTTAACTCCACCATT & 15,961-15,984\\
MAP001-R   & 58.9 & (M13-Rev\footnote{`M13-Rev' = CAGGAAACAGCTATGAC})TACTGCGACATAGGGTGCTC & 107-126\\
MAP002.1-F & 59.3 & (-21M13-Fwd)GAGCTCTCCATGCATTTGG &   36-54\\
MAP002.1-R & 57.3 & (M13-Rev)AGGGTGAACTCACTGGAACG   & 707-726\\
\end{tabular}}
\end{minipage}
\end{table}

\subsection{Construction of DNA Pools}\label{var.methods.pools}
DNA products from long-PCR were quantitated with Quant-iT\texttrademark PicoGreen\textsuperscript{\textregistered} reagent (Invitrogen).  Two DNA pools were constructed. One pool consisted of 10 ng mtDNA from each of 419 case samples, and the other consisted of 10 ng mtDNA from each of 415 control samples.  DNA was concentrated by speed-vac.

\subsection{Library Construction and Sequencing}\label{var_detect.methods.library}
Library construction and DNA sequencing was carried out by the sequencing platform of the BC Genome Sciences Centre.  Pooled mtDNA was sheared using sonication and size-separated using electrophoresis. The $\sim300$-bp fraction was isolated for library construction using the Illumina Genome Analyzer single-end library protocol (Illumina). Sequencing was performed on an Illumina GA using two lanes of a flow cell per pool, generating 36-bp reads.

\begin{table}[htbp]
\begin{minipage}{\textwidth}
\caption{Summary of Illumina Sequence Mapping (Untrimmed Reads)}
\label{var.table.illumina.untrimmed}
\noindent\makebox[\textwidth]{%
\footnotesize
\begin{tabular}{l r r l r l}
Chr.           & Length      & \multicolumn{2}{c}{Reads Mapped, Cases} & \multicolumn{2}{c}{Reads Mapped, Controls} \\ \hline 
 1             & 249,250,621 &    512,872 &   (4.30\%) &    539,464 &   (4.23\%) \\ 
 2             & 243,199,373 &     81,462 &   (0.68\%) &     85,478 &   (0.67\%) \\ 
 3             & 198,022,430 &     74,357 &   (0.62\%) &     77,525 &   (0.61\%) \\ 
 4             & 191,154,276 &     48,754 &   (0.41\%) &     53,598 &   (0.42\%) \\ 
 5             & 180,915,260 &    206,336 &   (1.73\%) &    225,336 &   (1.77\%) \\ 
 6             & 171,115,067 &     32,858 &   (0.28\%) &     35,324 &   (0.28\%) \\ 
 7             & 159,138,663 &    170,779 &   (1.43\%) &    227,925 &   (1.79\%) \\ 
 8             & 146,364,022 &     35,068 &   (0.29\%) &     38,300 &   (0.30\%) \\ 
 9             & 141,213,431 &     27,073 &   (0.23\%) &     26,942 &   (0.21\%) \\ 
10             & 135,534,747 &     25,878 &   (0.22\%) &     25,809 &   (0.20\%) \\ 
11             & 135,006,516 &     97,174 &   (0.81\%) &     98,442 &   (0.77\%) \\ 
12             & 133,851,895 &     28,901 &   (0.24\%) &     30,747 &   (0.24\%) \\ 
13             & 115,169,878 &     35,384 &   (0.30\%) &     36,679 &   (0.29\%) \\ 
14             & 107,349,540 &     23,239 &   (0.19\%) &     24,482 &   (0.19\%) \\ 
15             & 102,531,392 &     13,155 &   (0.11\%) &     14,121 &   (0.11\%) \\ 
16             &  90,354,753 &     13,580 &   (0.11\%) &     13,829 &   (0.11\%) \\ 
17             &  81,195,210 &    311,656 &   (2.61\%) &    299,374 &   (2.35\%) \\ 
18             &  78,077,248 &     19,447 &   (0.16\%) &     20,532 &   (0.16\%) \\ 
19             &  59,128,983 &      7,709 &   (0.06\%) &      7,346 &   (0.06\%) \\ 
20             &  63,025,520 &     11,142 &   (0.09\%) &     11,389 &   (0.09\%) \\ 
21             &  48,129,895 &     13,807 &   (0.12\%) &     14,062 &   (0.11\%) \\ 
22             &  51,304,566 &      5,967 &   (0.05\%) &      5,941 &   (0.05\%) \\ 
 X             & 155,270,560 &     54,631 &   (0.46\%) &     62,053 &   (0.49\%) \\ 
 Y             &  59,373,566 &     11,447 &   (0.10\%) &     11,022 &   (0.09\%) \\ 
MT             &      16,569 &  4,286,809 &  (35.94\%) &  4,681,659 &  (36.68\%) \\ 
other          &   6,110,758 &      5,068 &   (0.04\%) &      5,087 &   (0.04\%) \\ \hline
total mapped   & -           &  6,154,553 &  (51.59\%) &  6,672,466 &  (52.27\%) \\ 
total unmapped & -           &  5,774,653 &  (48.41\%) &  6,092,354 &  (47.73\%) \\ 
grand total    & -           & 11,929,206 & (100.00\%) & 12,764,820 & (100.00\%) \\
\end{tabular}}
\end{minipage}
\end{table}

\begin{table}[htbp]
\begin{minipage}{\textwidth}
\caption{Summary of Illumina Sequence Mapping (Trimmed Reads)}
\label{var.table.illumina.trimmed}
\noindent\makebox[\textwidth]{%
\footnotesize
\begin{tabular}{l r r l r l}
Chr. &      Length & \multicolumn{2}{c}{Reads Mapped, Cases} & \multicolumn{2}{c}{Reads Mapped, Controls} \\ \hline
 1 &   249,250,621 &   496,262 &   (6.92\%) &     508,839 &   (6.92\%) \\ 
 2 &   243,199,373 &    95,787 &   (1.34\%) &      94,431 &   (1.28\%) \\ 
 3 &   198,022,430 &    86,431 &   (1.20\%) &      82,319 &   (1.12\%) \\ 
 4 &   191,154,276 &    64,637 &   (0.90\%) &      65,229 &   (0.89\%) \\ 
 5 &   180,915,260 &   263,349 &   (3.67\%) &     270,725 &   (3.68\%) \\ 
 6 &   171,115,067 &    38,931 &   (0.54\%) &      37,264 &   (0.51\%) \\ 
 7 &   159,138,663 &   112,712 &   (1.57\%) &     107,757 &   (1.47\%) \\ 
 8 &   146,364,022 &    43,377 &   (0.60\%) &      42,340 &   (0.58\%) \\ 
 9 &   141,213,431 &    35,488 &   (0.49\%) &      32,654 &   (0.44\%) \\ 
10 &   135,534,747 &    32,305 &   (0.45\%) &      29,541 &   (0.40\%) \\ 
11 &   135,006,516 &   101,180 &   (1.41\%) &      96,423 &   (1.31\%) \\ 
12 &   133,851,895 &    35,422 &   (0.49\%) &      35,238 &   (0.48\%) \\ 
13 &   115,169,878 &    36,647 &   (0.51\%) &      36,389 &   (0.50\%) \\ 
14 &   107,349,540 &    29,971 &   (0.42\%) &      28,723 &   (0.39\%) \\ 
15 &   102,531,392 &    16,489 &   (0.23\%) &      15,896 &   (0.22\%) \\ 
16 &    90,354,753 &    17,579 &   (0.25\%) &      16,484 &   (0.22\%) \\ 
17 &    81,195,210 &   392,551 &   (5.47\%) &     340,912 &   (4.64\%) \\ 
18 &    78,077,248 &    22,205 &   (0.31\%) &      21,735 &   (0.30\%) \\ 
19 &    59,128,983 &     8,908 &   (0.12\%) &       7,365 &   (0.10\%) \\ 
20 &    63,025,520 &    14,805 &   (0.21\%) &      13,690 &   (0.19\%) \\ 
21 &    48,129,895 &    16,454 &   (0.23\%) &      14,409 &   (0.20\%) \\ 
22 &    51,304,566 &     7,768 &   (0.11\%) &       6,995 &   (0.10\%) \\ 
 X &   155,270,560 &    66,354 &   (0.93\%) &      64,348 &   (0.88\%) \\ 
 Y &    59,373,566 &    11,477 &   (0.16\%) &      10,403 &   (0.14\%) \\ 
MT &        16,569 & 3,750,715 &  (52.29\%) &   4,015,230 &  (54.64\%) \\ 
other &  6,110,758 &     5,121 &   (0.07\%) &       4,722 &   (0.06\%) \\ \hline 
  total mapped & - & 5,802,925 &  (80.90\%) &   6,000,061 &  (81.64\%) \\ 
total unmapped & - & 1,370,272 &  (19.10\%) &   1,348,969 &  (18.36\%) \\ 
   grand total & - & 7,173,197 & (100.00\%) &   7,349,030 & (100.00\%) \\
\end{tabular}}
\end{minipage}
\end{table}

\subsection{Statistical Analysis}\label{var.methods.statistical_analysis}
In order to assess the effect of read trimming on mapping, alignments were done with both full 36-base reads and trimmed reads.  For trimmed reads, the BWA read-trimming parameter (q=25) was used.  Short sequence reads were aligned to the GRCh37 (hg19) reference using the BWA sequence alignment program, version 0.6.1-r104\cite{li2009fast}.  Aside from the read-trimming parameter, all reads were mapped using default BWA parameters.

Per-base quality scores for both untrimmed and trimmed reads were calculated with FastQC software\cite{andrews2010fastqc} (See figures ~\ref{var.fig.base_quality_case}, ~\ref{var.fig.base_quality_cont})

SNPs were detected by analyzing BWA `pileup' output files with a custom perl script. At each position, the numbers of reference and non-reference bases were counted.  Only those bases with phred-scaled quality scores of 40 were included for SNP detection.

\section{Results}\label{var.results}

\subsection{Sequencing of the Mitochondrial Conrol Region}

The highly polymorphic mitochondrial control region rCRS (position 16024-576) was sequenced using bi-directional Sanger sequencing.  We discovered 277 SNPs in the control region that were present in at least one sample.

\begin{figure}
  \begin{center}
    \subfloat[Alignment]{\includegraphics[width=0.85\textwidth]{fig/consed.png}}\\
    \subfloat[Traces]{\includegraphics[width=0.85\textwidth]{fig/consed_trace.png}}  \end{center}
  \caption[Screenshots for Variant Discovery with Phred/Phrap/Consed]{
    \small{\textbf{SNP Calling by Phred/Phrap/Consed + PolyPhred.} (\textbf{a}) Reads were aligned to the revised Cambridge Reference Sequence (\ac{rCRS}).  Each sample was sequenced in both forward and reverse directions. Only a subset of samples are shown Sample IDs are at left in yellow type.  (\textbf{b}) Variants were identified automatically using PolyPhred, and confirmed manually by visual inspection of sequence traces. Two samples (127\_WIL and 128\_SIN) with differing alleles at contig position 1,288 (rCRS position 16,288) are shown.}}
  \label{var.fig.consed}
\end{figure}

\begin{figure}[htpb]
  \begin{center}
    \subfloat[a]{\includegraphics[width=\textwidth]{fig/heteroplasmy_01.png}}\\
    \subfloat[b]{\includegraphics[width=\textwidth]{fig/heteroplasmy_02.png}}  \end{center}
  \caption[Putative Heteroplasmic Positions]{
    \small{\textbf{Putative Heteroplasmic Positions.} Heteroplasmy was observed in some samples by identifying double-peaks in sequence traces. (\textbf{a}) Sample `157\_EPP' shows putative heteroplasmy level of $\sim 25\%$ at contig position 1,189 (rCRS position 16,189). (\textbf{b}) Sample `489\_SAM' shows putative heteroplasmy level of $\sim 50\%$ at contig position 1,126 (rCRS position 16,126).} Note that the relative heights of the two peaks at the heteroplasmic positions are consistent in forward and reverse reads.}
  \label{var.fig.heteroplasmy}
\end{figure}

\subsection{Next-Generation Sequencing of Pooled mtDNA}\label{var.results.illumina_sequencing}
The median depth of sequence coverage was 31,134 reads for the case pool and 12,683 reads for the control pool.  This represents approximately 30x coverage for each sample that is included in the pool.  To reduce the number of false-positive variant calls that are due to sequencing errors, we only considered high-quality bases (base-quality score $> 35$ and mapping-qualty score $> 20$) for SNP-calling.  We identified 90 SNPs in the case pool and 113 SNPs in the control pool with MAF $> 1\%$. 84 of these SNPs are common to both pools, with 6 SNPs only being observed in the case pool and 29 SNPs only being observed in the control pool. (see figs ~\ref{var.fig.snp_case} and ~\ref{var.fig.snp_cont}).

Comparison of minor allele frequencies for control region SNPs in Sanger and Illumina GA datasets is shown in Figures ~\ref{var.fig.comparison_case} and ~\ref{var.fig.comparison_cont}.  We found close correlation (Spearman's $r = 0.88$ in cases, Spearman's $r = 0.88$ in controls, $N = 277$).  We also observed that pooled Illumina GA sequencing produced consistently lower MAF estimates than Sanger sequencing of individual samples in this region.

\begin{figure}
  \begin{center}
  \subfloat[Case Pool, Untrimmed]{\includegraphics[width=0.75\textwidth]{fig/hs0297_case_per_base_quality_untrimmed.png}} \\
  \subfloat[Case Pool, Trimmed]{\includegraphics[width=0.75\textwidth]{fig/hs0297_case_per_base_quality_trimmed.png}}
  \end{center}
  \caption[Per-base Quality Distributions]{
    \small{\textbf{Effect of Read-trimming on Per-base Quality Distributions (Case Pool)} Average base quality score was calculated at each read position, across all reads.  For each position, red line indicates median quality score, yellow box indicates interquartile range (25\--75\%), upper and lower whiskers represent 90\% and 10\% quantiles, respectively, and blue line represents mean quality score. The upwards shift in average quality for trimmed reads indicates that poor-qualty sequence near the 3' end of reads has been removed in trimmed reads. }}
  \label{var.fig.base_quality_case}
\end{figure}

\begin{figure}
  \begin{center}
  \subfloat[Control Pool, Untrimmed]{\includegraphics[width=0.75\textwidth]{fig/hs0298_cont_per_base_quality_untrimmed.png}} \\
  \subfloat[Control Pool, Trimmed]{\includegraphics[width=0.75\textwidth]{fig/hs0298_cont_per_base_quality_trimmed.png}}
  \end{center}
  \caption[Per-base Quality Distributions]{
    \small{\textbf{Effect of Read-trimming on Per-base Quality Distributions (Control Pool)} Average base quality score was calculated at each read position, across all reads.  For each position, red line indicates median quality score, yellow box indicates interquartile range (25\--75\%), upper and lower whiskers represent 90\% and 10\% quantiles, respectively, and blue line represents mean quality score.  The upwards shift in average quality for trimmed reads indicates that poor-qualty sequence near the 3' end of reads has been removed in trimmed reads.}}
  \label{var.fig.base_quality_cont}
\end{figure}

\begin{figure}
\noindent\makebox[\textwidth]{%
  \includegraphics[width=1.4\textwidth]{fig/coverage_case.png}}
  \caption[Sequence Coverage for Case Pool]{
    \small{\textbf{Sequence coverage across the mitochondrial genome (Case Pool).} Blue line indicates high-quality (phred-scaled quality score = 40) sequence coverage. Graph lines every 10,000-fold depth.}}
  \label{var.fig.coverage_case}
\end{figure}

\begin{figure}
\noindent\makebox[\textwidth]{%
  \includegraphics[width=1.4\textwidth]{fig/coverage_cont.png}}
  \caption[Sequence Coverage for Control Pool]{
    \small{\textbf{Sequence coverage across the mitochondrial genome (Control Pool).} Blue line indicates high-quality (phred-scaled quality score = 40) sequence coverage. Graph lines every 10,000-fold depth.}}
  \label{var.fig.coverage_cont}
\end{figure}

\begin{figure}
\noindent\makebox[\textwidth]{%
  \includegraphics[width=1.4\textwidth]{fig/illumina_snps_case.png}}
  \caption[Minor Allele Frequencies for Case Pool]{
    \small{\textbf{Minor allele frequencies (Case Pool).} Locations and minor allele frequencies for all SNPs detected by Illumina GA sequencing.  Base identities are indicated as follows: A = Red, C = Blue, G = Orange, T = Green.  Heights of data bars indicate minor allele frequencies, scale bars every 10\% allele frequency.}}
  \label{var.fig.snp_case}
\end{figure}

\begin{figure}
\noindent\makebox[\textwidth]{%
  \includegraphics[width=1.4\textwidth]{fig/illumina_snps_cont.png}}
  \caption[Minor Allele Frequencies for Control Pool]{
    \small{\textbf{Minor allele frequencies (Control Pool).} Locations and minor allele frequencies for all SNPs detected by Illumina GA sequencing.  Base identities are indicated as follows: A = Red, C = Blue, G = Orange, T = Green.  Heights of data bars indicate minor allele frequencies, scale bars every 10\% allele frequency.}}
  \label{var.fig.snp_cont}
\end{figure}

\begin{figure}
\noindent\makebox[\textwidth]{%
  \includegraphics[width=\textwidth]{fig/solexa_sanger_maf_comparison_case.png}}
  \caption[MAF Comparison for Cases]{
    \small{\textbf{MAF comparison (Cases).}  Minor allele frequencies were determined by both Sanger sequencing and by pooled Illumina GA sequencing for 277 SNPs in the control region.  The Spearman's rank correlation between the two estimates is 0.88 in the case sample set, and 0.91 in controls.  Dashed line indicates slope = 1; the least-squares regression line is indicated by a solid line.}}
  \label{var.fig.comparison_case}
\end{figure}

\begin{figure}
\noindent\makebox[\textwidth]{%
  \includegraphics[width=\textwidth]{fig/solexa_sanger_maf_comparison_cont.png}}
  \caption[MAF Comparison for Controls]{
    \small{\textbf{MAF comparison (Controls).}  Minor allele frequencies were determined by both Sanger sequencing and by pooled Illumina GA sequencing for 277 SNPs in the control region.  The Spearman's rank correlation between the two estimates is 0.91 in control sample set.  Dashed line indicates slope = 1; the least-squares regression line is indicated by a solid line.}}
  \label{var.fig.comparison_cont}
\end{figure}

\begin{table}[htbp]
\begin{minipage}{\textwidth}
\caption[Mitochondrial Non-synonymous SNPs]{Functional Consequences for Non-synonymous SNPs}
\label{var.table.non-synon}
\noindent\makebox[\textwidth]{%
\footnotesize
\begin{tabular}{l r c c l l l}
ID           & Position & MAF (Seniors) & MAF(Controls) & Gene & Amino Acid Change & PolyPhen  \\ \hline
rs28357980   &   4,917  &         0.073 &         0.060 &  MT-ND2 & N [Asn] $\Rightarrow$ D [Asp] & 0.129 (benign) \\
rs28358886   &   8,697  &         0.078 &         0.053 & MT-ATP6 & M [Met] $\Rightarrow$ I [Ile] & 0.890 (possibly damaging) \\
rs9645429    &   9,055  &         0.070 &         0.051 & MT-ATP6 & A [Ala] $\Rightarrow$ T [Thr] & 0.845 (possibly damaging) \\
rs2853826    &  10,398  &         0.055 &         0.059 &  MT-ND3 & T [Thr] $\Rightarrow$ A [Ala] & 0.000 (benign) \\
rs28359178   &  13,708  &         0.041 &         0.028 &  MT-ND5 & A [Ala] $\Rightarrow$ T [Thr] & 0.000 (benign) \\
rs3135031    &  14,766  &         0.084 &         0.072 & MT-CYTB & T [Thr] $\Rightarrow$ I [Ile] & 0.000 (benign) \\
rs28357681   &  14,798  &         0.109 &         0.077 & MT-CYTB & F [Phe] $\Rightarrow$ L [Leu] & 0.000 (benign) \\
rs2853508    &  15,326  &         0.245 &         0.218 & MT-CYTB & T [Thr] $\Rightarrow$ A [Ala] & 0.000 (benign) \\
rs3088309    &  15,452  &         0.134 &         0.118 & MT-CYTB & L [Leu] $\Rightarrow$ I [Ile] & 0.029 (benign)      
\end{tabular}}
\end{minipage}
\end{table}

\begin{table}[htbp]
\begin{minipage}{\textwidth}
\caption[Number of Variants by Gene]{Number of Variants by Gene}
\label{var.table.num_variants_by_gene}
\noindent\makebox[\textwidth]{%
\footnotesize
\begin{tabular}{l r r r r r}
         &           & \multicolumn{2}{c}{Total Variants} & \multicolumn{2}{c}{Variants per kb}\\  
Gene     & Size (bp) & Cases & Controls & Cases & Controls\\ \hline
MT-TF    &    71 &  0 &  0 &  0.0 &  0.0 \\ 
MT-RNR1  &   954 &  7 & 10 &  7.3 & 10.5 \\ 
MT-TV    &    69 &  0 &  0 &  0.0 &  0.0 \\ 
MT-RNR2  & 1,559 & 17 & 16 & 10.9 & 10.3 \\ 
MT-TL1   &    75 &  0 &  0 &  0.0 &  0.0 \\ 
MT-ND1   &   956 &  8 &  7 &  8.4 &  7.3 \\ 
MT-TI    &    69 &  0 &  0 &  0.0 &  0.0 \\ 
MT-TQ    &    72 &  1 &  1 & 13.9 & 13.9 \\ 
MT-TM    &    68 &  0 &  0 &  0.0 &  0.0 \\ 
MT-ND2   & 1,042 & 12 & 19 & 11.5 & 18.2 \\ 
MT-TW    &    68 &  0 &  0 &  0.0 &  0.0 \\ 
MT-TA    &    69 &  1 &  2 & 14.5 & 29.0 \\ 
MT-TN    &    73 &  0 &  0 &  0.0 &  0.0 \\ 
MT-TC    &    66 &  0 &  1 &  0.0 & 15.2 \\ 
MT-TY    &    66 &  0 &  0 &  0.0 &  0.0 \\ 
MT-CO1   & 1,542 & 11 & 16 &  7.1 & 10.4 \\ 
MT-TS1   &    69 &  1 &  1 & 14.5 & 14.5 \\ 
MT-TD    &    68 &  0 &  0 &  0.0 &  0.0 \\ 
MT-CO2   &   684 &  3 &  4 &  4.4 &  5.8 \\ 
MT-TK    &    70 &  0 &  1 &  0.0 & 14.3 \\ 
MT-ATP8  &   207 &  3 &  3 & 14.5 & 14.5 \\ 
MT-ATP6  &   681 &  7 &  9 & 10.3 & 13.2 \\ 
MT-C03   &   784 & 10 &  9 & 12.8 & 11.5 \\ 
MT-TG    &    68 &  1 &  1 & 14.7 & 14.7 \\ 
MT-ND3   &   346 &  7 &  6 & 20.2 & 17.3 \\ 
MT-TR    &    65 &  1 &  1 & 15.4 & 15.4 \\ 
MT-ND4L  &   297 &  2 &  3 &  6.7 & 10.1 \\ 
MT-ND4   & 1,378 & 18 & 24 & 13.1 & 17.4 \\ 
MT-TH    &    69 &  0 &  0 &  0.0 &  0.0 \\ 
MT-TS2   &    59 &  0 &  0 &  0.0 &  0.0 \\ 
MT-TL2   &    71 &  1 &  2 & 14.1 & 28.2 \\ 
MT-ND5   & 1,812 & 28 & 27 & 15.5 & 14.9 \\ 
MT-ND6   &   525 & 10 &  6 & 19.0 & 11.4 \\ 
MT-TE    &    69 &  0 &  0 &  0.0 &  0.0 \\ 
MT-CYTB  & 1,141 & 20 & 22 & 17.5 & 19.3 \\ 
MT-TT    &    66 &  3 &  5 & 45.5 & 75.8 \\ 
MT-TP    &    68 &  0 &  0 &  0.0 &  0.0 \\ \hline
All Protein-coding & 11,395 & 139 & 155 & 12.2 & 13.6 \\ 
All RNA-coding     &  4,021 &  33 &  41 &  8.2 & 10.2 \\  
\end{tabular}}
\end{minipage}
\end{table}

For each gene in the mitochondrial genome, the number of variants observed at $\geq 1\%$ frequency were tabulated (Table \ref{var.table.num_variants_by_gene}). The most variable protein-coding gene is MT-ND3, with 20.2 variants per \ac{kb} in cases, and 17.3 variants per \ac{kb} in controls.  The most variable RNA gene is MT-TT, with 45.5 variants per \ac{kb} in cases and 75.5 variants per \ac{kb} in controls.

\section{Discussion}
We have shown here that it is possible to discover variants across the entire mitochondrial genome in over 400 samples in a single sequencing experiment. By combining long-PCR with second-generation sequencing technology, we were able to estimate the alllele frequencies of over 300 mitochondrial SNPs in our study population. This technique will be useful for rapidly surveying a large sample set for mitochondrial SNPs. Given its small size and high copy number per cell, mtDNA is a good candidate for pooled targeted resequencing efforts. The size of the mitochondrial chromosome (16.5 \ac{kb}) makes it amenable to long PCR. The whole mtDNA genome can be amplified in one reaction, which simplifies the DNA pooling process.  A similar variant detection has been employed by another group, using a pool size of 20 samples\cite{wang2011estimating}.

Figures \ref{var.fig.coverage_case} and \ref{var.fig.coverage_cont} show that the entire mitochondrial genome was sufficiently covered by mapped reads to perform variant detection.  There are strong peaks in coverage in both the case pool and control pool near position 200 within the control region.  We attribute this peak to excess PCR primers that were carried through into the sequencing reaction.

A previous report showed accurate determination of allele frequencies of pooled genomic DNA on the ABI SOLiD, Roche 454 and Illumina GA II platforms \cite{druley2009quantification}. Our estimation of MAF from Illumina sequencing of DNA pools correlates strongly with MAF calculated using genotypes determined using Sanger sequence data (Spearman's $r$ = 0.88); this correlation is close to the value of $r^{2} = 0.9637$ published by Druley et al\cite{druley2009quantification}. The most likely source of discrepancy between these two datasets is due to small differences in the quantity of DNA that each sample contributes to the DNA pool. 

In our analyses, MAF estimated from Illumina GA data is about 25\% lower than our measurement from Sanger sequencing. We suggest that this discrepancy may represent a bias against mapping of reads containing non-reference bases. We suggest that a read that contains a real non-reference base in the form of a SNP is less likely to align than a read that contains no non-reference SNPs, and that this probem will be increased in low-quality sequence data.  This phenomenon, referred to as `reference bias,' has been observed in previous studies of next-generation sequence data\cite{degner2009effect}.

The number of variants observed in at least 1\% of samples varied from 0 (MT-TY, MT-TF for example) to 27 (MT-ND5) (see table ~\ref{var.table.num_variants_by_gene}).  When normalized by the length of the gene, the most variable genes are MT-ND3 (20.2 variants/kb in cases, 17.3 variants/kb in controls) and MT-TT (45.5 variants/kb in cases, 75.8 variants/kb in controls).  Note, however, that the short length of the tRNA genes ($\sim 70$ bp) leads to a highly variable estimate of variants/kb.  Overall the distribution of variants was similar in protien-coding and RNA-coding genes at roughly 10 variants/kb.

Although our study was not designed to investigate the role that heteroplasmic variants play in the aging process, we did detect a small number of putative heteroplasmic variants by Sanger sequencing.  For low levels of heteroplasmy, (below $\sim 25\%$) it would be difficult to distinguish a true heteroplasmic variant from background noise in the sequence trace.  The few instances of heteroplasmy that we were able to identify with some certainty appeared to be close to 50\% heteroplasmic (See ~\ref{var.fig.heteroplasmy} for a representative example).

% Force a new page
\newpage
